table/shell/src/main/java/alluxio/cli/table/command/LoadTableCommand.java

```diff
@@ -0,0 +1,343 @@
+    }
+    mActiveJobs = FileSystemShellUtils.getIntArg(cl, ACTIVE_JOB_COUNT_OPTION,
+        AbstractDistributedJobCommand.DEFAULT_ACTIVE_JOBS);
+    System.out.format("Allow up to %s active jobs%n", mActiveJobs);
+    AlluxioConfiguration conf = mFsContext.getClusterConf();
+    int defaultBatchSize = conf.getInt(PropertyKey.JOB_REQUEST_BATCH_SIZE);
+    int replication = FileSystemShellUtils.getIntArg(cl, REPLICATION_OPTION, DEFAULT_REPLICATION);
+    int batchSize = FileSystemShellUtils.getIntArg(cl, BATCH_SIZE_OPTION, defaultBatchSize);
+    boolean directCache = !cl.hasOption(PASSIVE_CACHE_OPTION.getLongOpt());
+    Set<String> workerSet = new HashSet<>();
+    Set<String> excludedWorkerSet = new HashSet<>();
+    Set<String> localityIds = new HashSet<>();
+    Set<String> excludedLocalityIds = new HashSet<>();
+    if (cl.hasOption(HOST_FILE_OPTION.getLongOpt())) {
+      String hostFile = cl.getOptionValue(HOST_FILE_OPTION.getLongOpt()).trim();
+      readLinesToSet(workerSet, hostFile);
+    } else if (cl.hasOption(HOSTS_OPTION.getLongOpt())) {
+      String argOption = cl.getOptionValue(HOSTS_OPTION.getLongOpt()).trim();
+      readItemsFromOptionString(workerSet, argOption);
+    }
+    if (cl.hasOption(EXCLUDED_HOST_FILE_OPTION.getLongOpt())) {
+      String hostFile = cl.getOptionValue(EXCLUDED_HOST_FILE_OPTION.getLongOpt()).trim();
+      readLinesToSet(excludedWorkerSet, hostFile);
+    } else if (cl.hasOption(EXCLUDED_HOSTS_OPTION.getLongOpt())) {
+      String argOption = cl.getOptionValue(EXCLUDED_HOSTS_OPTION.getLongOpt()).trim();
+      readItemsFromOptionString(excludedWorkerSet, argOption);
+    }
+    if (cl.hasOption(LOCALITY_FILE_OPTION.getLongOpt())) {
+      String localityFile = cl.getOptionValue(LOCALITY_FILE_OPTION.getLongOpt()).trim();
+      readLinesToSet(localityIds, localityFile);
+    } else if (cl.hasOption(LOCALITY_OPTION.getLongOpt())) {
+      String argOption = cl.getOptionValue(LOCALITY_OPTION.getLongOpt()).trim();
+      readItemsFromOptionString(localityIds, argOption);
+    }
+    if (cl.hasOption(EXCLUDED_LOCALITY_FILE_OPTION.getLongOpt())) {
+      String localityFile = cl.getOptionValue(EXCLUDED_LOCALITY_FILE_OPTION.getLongOpt()).trim();
+      readLinesToSet(excludedLocalityIds, localityFile);
+    } else if (cl.hasOption(EXCLUDED_LOCALITY_OPTION.getLongOpt())) {
+      String argOption = cl.getOptionValue(EXCLUDED_LOCALITY_OPTION.getLongOpt()).trim();
+      readItemsFromOptionString(excludedLocalityIds, argOption);
+    }
+    List<URIStatus> filePool = new ArrayList<>(batchSize);
+    // Only support hive table for now.
+    String udbType = "hive";
+    AlluxioURI path = CatalogPathUtils.getTablePathUdb(dbName, tableName, udbType);
+    System.out.printf("Loading table %s.%s...%n", dbName, tableName);
+    DistributedLoadUtils.distributedLoad(this, filePool, batchSize, path, replication, workerSet,
+        excludedWorkerSet, localityIds, excludedLocalityIds, directCache, true);
```

Comment:
Does it seem to assume all partition paths are under the same table parent path? Is this statement always true?

Reply:
For now, `alluxio table attachdb` can not work-on tables which partition path in different parent path, this statement is true temporary. But when `alluxio table attachdb` support different parent path for partitions, this statement can not guaranteed to be correct. Is there any suggestions?

Reply:
For now, `alluxio table attachdb` can not work-on tables which partition path in different parent path, this statement is true temporary. But when `alluxio table attachdb` support different parent path for partitions, this statement can not guaranteed to be correct. Is there any suggestions?

Reply:
`CatalogPathUtils.getTablePathUdb` is able to get file path in alluxio which is mounted by alluxio from ufs when `alluxio table attachdb`

Reply:
It should be fine at this moment, can you add one line description to address the limitation?

Reply:
Sure~